packages/flutter/lib/src/widgets/drag_target.dart

```diff
@@ -383,6 +395,12 @@ class _DragTargetState<T> extends State<DragTarget<T>> {
   final List<_DragAvatar<T>> _candidateAvatars = <_DragAvatar<T>>[];
   final List<_DragAvatar<dynamic>> _rejectedAvatars = <_DragAvatar<dynamic>>[];
 
+  void onDrag(_DragAvatar<dynamic> avatar, Offset lastOffset) {
```

Comment:
I followed the didEnter(), didLeave()... pattern. Any idea why it's dynamic there? Change my method to T.